api/v1alpha1/openapi.yaml

```diff
@@ -2502,6 +2532,231 @@ components:
+        restarts:
+          type: integer
+          description: 'Number of restarts observed for the application.'
+    DeviceApplicationSummary:
+      type: object
+      properties:
+        status:
+          $ref: '#/components/schemas/DeviceApplicationSummaryType'
+          description: 'Summary status of applications.'
+        info:
+          type: string
+          description: 'Human readable information indicating details about the application status.'
+    DeviceApplicationSummaryType:
+      enum:
+      - 'Healthy'
+      - 'Degraded'
+      - 'Running'
+      - 'Error'
+      - 'Unknown'
+      x-enum-varnames:
+      - 'ApplicationsHealthy'
+      - 'ApplicationsDegraded'
+      - 'ApplicationsRunning'
+      - 'ApplicationsError'
+      - 'ApplicationsUnknown'
+    DeviceApplicationStatusType:
+      type: string
+      enum:
+      - 'Preparing'
+      - 'Starting'
+      - 'Running'
+      - 'Error'
+      - 'Unknown'
+      - 'Completed'
+      x-enum-varnames:
+      - 'ApplicationPreparing'
+      - 'ApplicationStarting'
+      - 'ApplicationRunning'
+      - 'ApplicationError'
+      - 'ApplicationUnknown'
+      - 'ApplicationCompleted'
+    DeviceRuntimeStatus:
+      type: object
+      properties:
+        status:
+          $ref: '#/components/schemas/DeviceRuntimeSummaryStatusType'
+          description: 'Status of the device runtime.'
+        info:
+          type: string
+          description: 'Human readable information indicating details about last transition.'
+    DeviceRuntimeSummaryStatusType:
+      type: string
+      enum:
+      - 'Online'
+      - 'Degraded'
+      - 'Error'
+      - 'Rebooting'
+      - 'PoweredOff'
+      - 'Unknown'
+      x-enum-varnames:
+      - 'DeviceRuntimeOnline'
+      - 'DeviceRuntimeDegraded'
+      - 'DeviceRuntimeError'
+      - 'DeviceRuntimeRebooting'
+      - 'DeviceRuntimePoweredOff'
+      - 'DeviceRuntimeUnknown'
+    DeviceUpdateStatus:
+      type: object
+      properties:
+        status:
+          $ref: '#/components/schemas/DeviceUpdateSummaryStatus'
+          description: 'Status of the device update.'
+        phase:
+          $ref: '#/components/schemas/DeviceUpdatePhaseStatus'
+          description: 'The passphrase for the private key.'
+    DeviceUpdateSummaryStatus:
+      type: object
+      properties:
+        status:
+          $ref: '#/components/schemas/DeviceUpdateStatusType'
+          description: 'Status summary type of the device update.'
+        info:
+          type: string
+          description: 'Human readable information about the last update transition.'
+    DeviceUpdateStatusType:
+      type: string
+      enum:
+      - 'UpToDate'
+      - 'OutOfDate'
+      - 'Updating'
+      - 'Unknown'
+      x-enum-varnames:
+      - 'DeviceUpdateUpToDate'
+      - 'DeviceUpdateOutOfDate'
+      - 'DeviceUpdateUpdating'
+      - 'DeviceUpdateUnknown'
+    DeviceUpdatePhaseStatus:
```

Comment:
The `updating` condition with the phase in `reason` was totally fine, I think (as we don't intend to filter by phase, only report in tooltip). The missing bit was the `DeviceUpdateStatus` summary status above that we filter/summarize.

Reply:
It would be good if you could update the DDD to reflect this or allow me edit access. Maybe a field that designates status vs condition. Then also sortable vs not to ensure we are all on the same page.

Reply:
removing